_posts/0000-01-02-Dinkleberg126.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Dogecoin (https://www.google.com/imgres?imgurl=https%3A%2F%2Fpyxis.nymag.com%2Fv1%2Fimgs%2F8f8%2Fe12%2F51b54d13d65d8ee3773ce32da03e1fa220-dogecoin.2x.rsocial.w600.jpg&imgrefurl=https%3A%2F%2Fnymag.com%2Fintelligencer%2F2021%2F04%2Fwhy-dogecoin-is-forcing-people-to-take-it-seriously.html&tbnid=wqMHZzk_nhRj7M&vet=12ahUKEwjr4bT0mZjxAhVRneAKHYxUAeQQMygAegUIARDSAQ..i&docid=k9kPMRxgg9oCUM&w=1200&h=632&q=Dogecoin&client=firefox-b-1-d&ved=2ahUKEwjr4bT0mZjxAhVRneAKHYxUAeQQMygAegUIARDSAQ)